### PR TITLE
feat(servicebus): add bodyAsText and parseBody helper utilities

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**
## Summary

Addresses #50 by adding utility functions for parsing Service Bus message bodies.

Since v0.4.0 (PR #36), `message.body` returns the raw `Buffer` rather than a pre-parsed object. This requires users to write boilerplate for string conversion and JSON parsing:

~~~typescript
const text = message.body instanceof Buffer ? message.body.toString('utf-8') : String(message.body);
const data = JSON.parse(text);
~~~

This PR adds two helper functions to reduce that friction:

- **`messageBodyAsText(message)`** - Convert message body to string (handles Buffer, string, or object)
- **`messageBodyAsJson<T>(message, reviver?)`** - Parse JSON body with optional reviver function

## Usage

~~~typescript
import { messageBodyAsJson, messageBodyAsText } from '@azure/functions-extensions-servicebus';

// Get body as string
const text = messageBodyAsText(message);

// Parse JSON body
const data = messageBodyAsJson<MyType>(message);

// Parse with custom reviver (e.g., Date conversion)
const data = messageBodyAsJson<MyType>(message, (key, value) => {
    if (key === 'timestamp' && typeof value === 'string') {
        return new Date(value);
    }
    return value;
});
~~~

## Design Decisions

- **Standalone utility functions** rather than instance methods - avoids SDK type modifications and works with any object that has a `body` property
- **Duck typing** - Functions accept `{ body: unknown }` for broad compatibility with different SDK versions
- **Explicit error handling** - Invalid JSON throws `SyntaxError`; invalid UTF-8 Buffer throws `TypeError` (uses `TextDecoder` with `fatal: true`)

## Testing

- 17 unit tests covering Buffer, string, object inputs, reviver functions, non-UTF-8 error handling, and edge cases
- All 231 unit tests passing
- E2E tests verified locally with Service Bus Emulator (4/4 passing)

## Note on Samples

The sample applications in this repository currently import from a local `servicebus-helpers.ts` file. Once this PR is released, samples can be updated to use the package exports directly. For local development, samples use `file:../../` package references.

### CODE REVIEW

File: azure-functions-nodejs-extensions-servicebus/src/util/bodyHelpers.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ * Returns the message body as a UTF-8 string.
+ * Handles Buffer, string, and object body types.
+ *
+ * @param message - A message object with a `body` property (e.g., ServiceBusReceivedMessage)
+ * @returns The body as a UTF-8 string
+ */
+export function bodyAsText(message: { body: unknown }): string {
+    const body: unknown = message.body;
+    if (Buffer.isBuffer(body)) {
+        return body.toString('utf8');
```
[the remark this answers is not in the document]
Comment (TsuyoshiUshio) [reply azure-functions-nodejs-extensions-servicebus/src/util/bodyHelpers.ts]: Updated. Custom will see exception. Check the unit testing. :)

**Review by swapnil-nagar — APPROVED** [review]
